examples/computer_vision/cifar10_tf_keras/cifar_model.py

```diff
@@ -6,11 +6,16 @@
 import numpy as np
 import tensorflow as tf
 from data import NUM_CLASSES
+from packaging import version
 from tensorflow.keras.layers import Activation, Conv2D, Dense, Dropout, Flatten, MaxPooling2D
 from tensorflow.keras.losses import categorical_crossentropy
 from tensorflow.keras.metrics import categorical_accuracy
 from tensorflow.keras.models import Sequential
-from tensorflow.keras.optimizers import RMSprop
+
+if version.parse("2.11.0") <= version.parse(tf.__version__):
+    from tensorflow.keras.optimizers.legacy import RMSprop
+else:
+    from tensorflow.keras.optimizers import RMSprop
```

Comment:
I like your suggestion to create a new ticket to migrate to the modern Keras interface. If you do, this is a good spot for a TODO(MLG-XXX):